app/components/elements/Login.jsx

```diff
@@ -1,29 +1,27 @@
 import React from "react";
 import PropTypes from "prop-types";
 import { Link } from "react-router-dom";
+import { compose } from "recompose";
+import { translate } from "react-i18next";
 
 import { postLogin } from "lib/api";
 import { errorTransform } from "lib/errorTransform";
 import FormError from "components/elements/FormError";
 
-import t from "translations/pt.yml";
+// import t from "translations/pt.yml";
 
 class Login extends React.Component {
   static contextTypes = {
     router: PropTypes.object,
     session: PropTypes.object
   }
 
-  constructor(...args) {
-    super(...args);
-
-    this.state = {
-      fields: {
-        email: "",
-        password: ""
-      },
-      errors: {}
-    };
+  state = {
+    fields: {
+      email: "",
+      password: ""
```

Comment:
Missing dangling comma. Should we configure eslint to force it?

Reply:
We definitely should